test.py

```diff
@@ -9,18 +9,26 @@
 import tempfile
 import time
 
-
```

Comment:
+1 to two line separation. Consistent style pays dividends, and while there can be cases where the tradeoffs make a style violation worth it, there's really no upshot to doing so here. Ideally we should just be running `flake8` against our Python code as a PR check, since enforcing style is mostly a job computers can do, saving reviewers' time for other concerns.